### PR TITLE
Added missing plugins folder to fix broken cmake-gui

### PR DESCRIPTION
**Pull request by willyd**
This fixes issue #12. The plugins folder is missing from the installation and this breaks cmake-gui.exe.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,9 +10,13 @@ source:
...
+  url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}-win32-x86.zip       # [win32]
+  fn: cmake-{{ version }}-win32-x86.zip                                                   # [win32]
+  sha256: 60089b4e05a997edda49f40d4b6db883e0ce5bbe9fdedb5106739dbfe58a8055                # [win32]
+
+  url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}-win64-x64.zip       # [win64]
+  fn: cmake-{{ version }}-win64-x64.zip                                                   # [win64]
+  sha256: c5f7bc95c395a5edae5a3311b844a275ae39db5a5572ac50da49d7496156f5f3                # [win64]
 
 build:
   number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please bump build number to `1`.

Comment (willyd) [inline recipe/meta.yaml]: Done.

**Comment by jakirkham** [comment]
Could you please review this, @msarahan?

**Comment by jakirkham** [comment]
Needs a rebase.

**Comment by scopatz** [comment]
This needs to be merged with master.

**Comment by pelson** [comment]
👍  in principle, but it doesn't merge cleanly, so will need rebasing.

**Comment by willyd** [comment]
Rebased on current master.

**Comment by jakirkham** [comment]
Seems this is failing. Also we need to [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) before proceeding now. Could we please put the re-rendering in a separate PR? That way we can better understand if this failure is in `master`.

**Comment by gqmelo** [comment]
Packaging these Qt5 files is a problem because it can overwrite files when someone has a Qt5 package (the defaults channel has Qt5 and there is intention to add it to conda-forge too). The Qt5 dlls were recently removed to fix that: https://github.com/conda-forge/cmake-feedstock/pull/26

So we cannot just add ```cmake-gui.exe``` and its dependencies to this package. If ```cmake-gui.exe``` is really desired I think the best way would be to build cmake from scratch on Windows too (and then maybe have a cmake-gui package which depends on qt5).

**Comment by jakirkham** [comment]
Agreed. Said as much [before]( https://github.com/conda-forge/cmake-feedstock/issues/12#issuecomment-246375830 ). Would prefer to close this out.

**Comment by jschueller** [comment]
onlu cli is provided

**Pull request closed without merging**